packages/tools/webpack-fluid-loader/src/multiDocumentServiceFactory.ts

```diff
@@ -26,12 +27,15 @@ export function getDocumentServiceFactory(documentId: string, options: RouteOpti
         name: getRandomName(),
     });
 
-    const routerliciousTokenProvider = new InsecureTokenProvider(
+    let routerliciousTokenProvider = new InsecureTokenProvider(
         (options as IRouterliciousRouteOptions).tenantId ,
         documentId,
         (options as IRouterliciousRouteOptions).tenantSecret,
         getUser());
 
+        if (options.mode === "tinylicious") {
```

Comment:
This should be formatted to the left.

Reply:
The formatting still looks wrong to me. Could be a github issue though!
Also, you should create the InsecureTokenProvider above as an else clause to this.
```
if (options.mode === "tinylicious") {
   // create the tinylicious one.
} else {
   // create the routerlicious one.
}
```

Reply:
Can you also add a comment here saying that the token provider and the routerlicious document service will not be called for local server and spo.